using-mobile-extensions/adobe-campaign-standard/adobe-campaign-standard-api-reference.md

```diff
@@ -226,3 +273,188 @@ ACPCore.setPushIdentifier("pushIdentifier");
 {% endtab %}
 {% endtabs %}
 
+## Tracking push or local notification message interactions
+
+User interactions with local or push notifications can be tracked by invoking the `collectMessageInfo` API. After the API is invoked, a network request will be made to Campaign containing the message interaction event. For more info on Campaign message interaction events, see [Implementing local notification tracking](https://helpx.adobe.com/campaign/kb/local-notification-tracking.html#Description). 
```

Comment:
- In the 2nd sentence, please replace "will be made" to "is made" and replace "containing" with "that contains".
- In the 3rd sentence, please replace "Info" with "information".